.github/workflows/manually-cancel-review.yaml

```diff
@@ -0,0 +1,19 @@
+on:
+    workflow_dispatch:
+        inputs:
+            branch_name:
+                description: 'Branch name'
+                required: true
+                type: string
+name: Manually cancel review
+jobs:
+    cancel-review:
+        if:  ${{ inputs.branch_name }} != 'master'
```

Comment:
Here you should probably somehow obtain the cancel-review.sh file

Reply:
fixed